pages/wards/visits/[id]/edit.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from "react";
+import { GridRow, GridColumn } from "../../../../src/components/Grid";
+import Layout from "../../../../src/components/Layout";
+import ErrorSummary from "../../../../src/components/ErrorSummary";
+import VisitForm from "../../../../src/components/VisitForm";
+import verifyToken from "../../../../src/usecases/verifyToken";
+import Router from "next/router";
+import propsWithContainer from "../../../../src/middleware/propsWithContainer";
+import { WARD_STAFF } from "../../../../src/helpers/userTypes";
+
+const EditAVisit = ({
+  callId,
+  initialPatientName,
+  initialContactName,
+  initialContactNumber,
+  initialContactEmail,
+  initialCallDateTime,
+}) => {
+  const [errors, setErrors] = useState([]);
+
+  const submit = () => {
+    Router.push({
+      pathname: `/wards/visits/${callId}/edit-confirmation`,
```

Comment:
Since this page doesn't exist yet, should we redirect to the success page for now?